das/metrics.go

```diff
@@ -76,11 +78,22 @@ func (d *DASer) InitMetrics() error {
 		return err
 	}
 
+	totalSampled, err := meter.
+		SyncInt64().
+		UpDownCounter(
```

Comment:
Consider using gauge. Counter will start from previous value upon restart, that will break observations